styles.css

```diff
@@ -1 +1,160 @@
+html {
+  height: 100%;
+}
 
+body {
+  background-color: #2B2733;
+  display:flex;
+  flex-direction:column;
```

Comment:
Gotta go through and standardize (lots of things don't have spaces, CSS could be better ordered).